src/services/member/member.controller.ts

```diff
@@ -49,7 +49,7 @@ const controller: FastifyPluginAsyncTypebox = async (fastify) => {
       if (user?.account) {
         const account = await accountRepository.get(db, user?.account?.id);
         // TODO: this type is wrong. Should accountRepository.get return either MemberDTO or GuestDTO (implementing AccountDTO)?
-        return account.toCurrent() as any;
+        return account.toCurrent() as never;
```

Comment:
Not sure about this one... I don't remember what the type issue was. Anyway between `any` and `never` 🤷

Reply:
I think a problem between ajv and having guest/member. The todo is still there, so I think it's okay.